design-tokens/web/scss/_texts.scss

```diff
@@ -0,0 +1,17 @@
+/**
+  text properties
+  eg.: font-size, line-height
+*/
+$font-properties: map-get($tokens, font);
+
+$font-sizes: map-get($font-properties, size);
+$line-height: map-get($font-properties, line-height);
+
+$keys: map-keys($font-sizes);
+
+@each $key in $keys {
+  .text-#{$key} {
```

Comment:
Here we called the classes by only `text-sm, text-md...` and on `_fonts.scss ` we are calling by `tec` prefix. 
What do you think if we remove the prefix?